Cargo.toml

```diff
@@ -26,6 +26,7 @@ codec = { package = "parity-scale-codec", version = "2", default-features = fals
 parking_lot = "0.11"
 
 [dev-dependencies]
+assert_matches = "1.5"
```

Comment:
Is this used anywhere?

Reply:
Oh right -- this was used in a test that I removed after changing my strategy for fixing this bug :) Removed dependency.